{ 
  "text" : "server/action.py in Fail2ban before 0.8.8 does not properly handle the content of the matches tag, which might allow remote attackers to trigger unsafe behavior in a custom action file via unspecified symbols in this content.\n" ,
  "entities" : [ { 
    "eid" : "E1" ,
    "keyid" : "DomainName#5aa1a000339e2b24d4c222f55fe929d7" ,
    "type" : "DomainName" ,
    "subType" : "DomainName" ,
    "value" : "action.py" ,
    "begin" : 7 ,
    "end" : 16 ,
    "canonicalForm" : "action.py" ,
    "score" : "0.0"
  }
  , { 
    "eid" : "E2" ,
    "keyid" : "9a0364b9e99bb480dd25e1f0284c8555" ,
    "type" : "MiscEntity" ,
    "subType" : "content" ,
    "value" : "content" ,
    "begin" : 71 ,
    "end" : 224 ,
    "canonicalForm" : "content" ,
    "score" : "0.0"
  }
  ,  ] ,
  "relations" : [ { 
    "rid" : "R1" ,
    "type" : "not_handle" ,
    "predicate" : { 
      "begin" : 42 ,
      "end" : 66 ,
      "surfaceForm" : "does not properly handle"
    }
     ,
    "arguments" : [ { 
      "argnum" : 1 ,
      "begin" : 7 ,
      "end" : 16 ,
      "eid" : "E1" ,
      "canonicalForm" : "action.py"
    }
    , { 
      "argnum" : 2 ,
      "begin" : 71 ,
      "end" : 224 ,
      "eid" : "E2" ,
      "canonicalForm" : "content"
    }
    ,  ]
  }
  ,  ]
}
